### DISCUSSION TITLE
Meetings where there is a specific name for the adress

### DISCUSSION THREAD

**Discussion by mikepoconnor4** [code4recovery/12-step-meeting-list#839] [Q&A] [answered]
I'm having a problem with certain meetings. We have many meetings held within a park or beach and the geocoding of the address sometimes doesn't show where the meeting is exactly.

For some of these meetings, people have developed a pin that appears to be registered with Google Maps, Waze, etc. And that pin shows exactly where the meeting is (I don't know who did that but somebody did it).

The problem comes in the feed between TSML and the Meeting Guide app. When I enter the specific pin name in TSML under WordPress - all is good, and the pin is dropped onto the exact spot and displays correctly on our website. However when it feeds over to the Meeting Guide app, I think what is happening is the specific pin is being lost, a generic geocoding occurs - and in our cases that address is actually quite far from the meeting spot - and then Meeting Guide is showing that "not great" geocoded address.

I don't know who exactly is responsible for what here. Is that more of a Meeting Guide issue and they need to allow these pin names through? or is that a TSML issue and it's dropping the pin name and doing the geocode and then feeding that along.

**Comment by joshreisner** [comment, marked as answer]
hi mike, how many of these are there? could you show us an example or two?

**Reply by mikepoconnor4** [reply]
There's maybe 5 to 10 - but one of them is famous and attracts lots of tourists, Twelve Coconuts, and I often hear people have a hard time finding the meeting (even with added verbiage to help them find it).

If you type in "Twelve Coconuts" in Google Maps you'll see the exact spot. See attachment for where you should find it. 

As I said when I type in "Twelve Coconuts" in the Address field on TSML the pin is placed properly - TSML looks like it geocodes at that point and the Location field becomes "Twelve Coconuts" and the Address becomes "2749 Kalakaua Ave, Honolulu, HI 96815, USA". The problem is that when it goes to Meeting Guide, the 2749 Kalakaua Ave address passes along and not Twelve Coconuts. If you search for that on Google Maps, it places the pin on the other side of a restaurant and people are looking in the wrong place for the meeting. (See second attched image).

To alleviate that, we use another address for that meeting - 2650 Kalakaua Ave (see third attachment). That keeps the website and Meeting Guide in sync - the problem is that address is across the street from the meeting location and there's a bandstand and a sizable park area and visitors are often wandering around there looking for the meeting - and sometimes they finally realize it's across the street - but I often hear they have a hard time finding the meeting.

![image](https://user-images.githubusercontent.com/74935325/175339925-04c40be5-330c-4139-bdc4-9009cbf32c01.png)

![image](https://user-images.githubusercontent.com/74935325/175341389-ac9e7b45-daa0-4494-9564-e07fd42bd212.png)

![image](https://user-images.githubusercontent.com/74935325/175342305-862a4b65-9e1c-4dbe-9340-b7b2cb54b441.png)

**Reply by joshreisner** [reply]
i see! how about `2679 Kalakaua Ave, Honolulu, HI 96815` -- that looks like it goes to the right place, more or less?
![Screen Shot 2022-06-23 at 9 24 44 AM](https://user-images.githubusercontent.com/1551689/175348166-2089f2e2-ed13-4260-bca1-7d3baae3142c.png)

**Reply by mikepoconnor4** [reply]
Oh that works. We've looked around for addresses and hadn't seen that one before - I wonder if that got added to some GPS database recently. How did you find that address?

When I get home, there's another one I'm having an issue with. Maybe you can spot that one too.

**Reply by joshreisner** [reply]
great! i just clicked on the bathroom and saw its address was `2669`, and since it's too far north and `2749` is too far south, i tried a number in between

**Reply by mikepoconnor4** [reply]
Actually the other one is working now - I think I didn't give Meeting Guide enough time to update when I originally looked (I had looked next day)- but it seems to have gone through with this funky coding. see https://oahuaa.org/oahu-meetings/?type=active&search=%28physical%29&view=map&meeting=physical-friendship-tree

The special map pin name was International Friendship Tree - which geocoded to this strange address - "75P3+CV International Friendship Tree, Honolulu, HI 96815, USA". 

And I have one other but I need to research that one more.

**Reply by mikepoconnor4** [reply]
You wouldn't believe some of the nastygrams we get when we can't get that pin down right.

**Reply by joshreisner** [reply]
great! and i very much would believe it :)
